tools/bin/ariaDownload.py

```diff
@@ -62,28 +65,28 @@ def __init__(self, inps):
         self.inps.output = self.inps.output.title()
         self.inps.wd     = op.abspath(self.inps.wd)
         self.url_base    = 'https://api.daac.asf.alaska.edu/services/search/param?'
+        if self.inps.v: log.setLevel('DEBUG')
+        return
 
     def __call__(self):
         url              = self.form_url()
         dict_prod, urls  = self.parse_json(url)
         script           = requests.post(f'{self.url_base}&output={self.inps.output}', data=dict_prod).text
 
-
         if self.inps.output == 'Count':
-            print (f'\nFound -- {len(urls)} -- products')
-
+            log.info('\nFound -- %d -- products', len(urls))
 
         elif self.inps.output == 'Kml':
             os.makedirs(self.inps.wd, exist_ok=True)
             dst = self._fmt_dst()
-            print (script, file=open(dst, 'w'))
-            print (f'Wrote .KMZ to:\n\t {dst}')
+            log.info(script, file=open(dst, 'w'))
+            log.info(f'Wrote .KMZ to:\n\t %s', dst)
```

Comment:
Codacy found an issue: [f-string is missing placeholders (F541)](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=5957665)